handler_test.go

```diff
@@ -111,15 +97,15 @@ func TestRouteProxyWithURI(t *testing.T) {
 	assert.Equal(t, 200, resp.StatusCode)
```

Comment:
That test is only used to check if the "proxy" works, by hitting an known resource (the agent's WebID profile).